### ISSUE TITLE
Rectangle not working seems due to accessibility element not given

### ISSUE THREAD

**Issue by zhast460** [rxhanson/Rectangle#1461]
macOS version: 14.6.1 (23G93)
Rectangle version: v0.82(88)
Logs if applicable (In Rectangle menu, hold option, "View Logging..."): plenty of:
Unable to obtain the accessibility element with the specified attribute at mouse location

However the accessibility is given in the system settings
<img width="715" alt="image" src="https://github.com/user-attachments/assets/d1411acc-7dad-4cce-868e-f94fef2664a3">

**Comment by rxhanson** [comment]
Indeed, this points to the accessibility permissions being out of sync. To get it back in sync

1. Close Rectangle if it's running
2. Here in System Settings*, first disable Rectangle, then remove it with the minus button. (it's important to do both of those steps in that order)
3. Launch Rectangle and enable settings for it as prompted.

If you're handy with the Terminal you can execute this terminal command instead of step 2:
```bash
tccutil reset All com.knollsoft.Rectangle
```

If it's still out of sync, you might have to logout/login.

\* System Settings > Privacy & Security > Accessibility
